source/game/field/obj/ObjectSandcone.hh

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "game/field/obj/ObjectKCL.hh"
+
+#include <optional>
```

Comment:
Unused?

Reply:
Ohhh yeah I moved something around and forgot to get rid of this